node/node.go

```diff
@@ -77,7 +77,7 @@ func NewNode(genDoc *genesis.Genesis, conf *config.Config,
 	}
 
 	consMgr := consensus.NewManager(conf.Consensus, st, valKeys, rewardAddrs, messageCh)
-	walletMgr := wallet.NewWalletManager(chainType)
+	walletMgr := wallet.NewWalletManager(chainType, conf.GRPC.WalletsDir)
```

Comment:
Not a good way to pass the wallet directory to the Wallet module. 
This configuration should be removed from the gRPC module since it is unused. 
We practice keeping each module's configuration isolated from others. 
Let's retrieve the wallet directory from the node initializer.